libs/game-state/src/lib/services/cards/razzle-dazzler.ts

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-mixed-spaces-and-tabs */
+import { CardIds, CardType } from '@firestone-hs/reference-data';
+import { GuessedInfo } from '../../models/deck-card';
+import { canBeDiscoveredByClass, hasCorrectType, hasCost } from '../../related-cards/dynamic-pools';
+import { GeneratingCard, GuessInfoInput, StaticGeneratingCard, StaticGeneratingCardInput } from './_card.type';
+import { filterCards } from './utils';
+
+// Razzle-Dazzler - Battlecry: Summon a random 5-Cost minion. Repeat for each spell school you've cast this game.
+export const RazzleDazzler: GeneratingCard & StaticGeneratingCard = {
+	cardIds: [CardIds.RazzleDazzler_VAC_301],
+	publicCreator: true,
+	dynamicPool: (input: StaticGeneratingCardInput) => {
+		return filterCards(
+			RazzleDazzler.cardIds[0],
+			input.allCards,
+			(c) =>
+				hasCorrectType(c, CardType.MINION) &&
+				hasCost(c, '==', 5) &&
+				canBeDiscoveredByClass(c, input.inputOptions.currentClass),
+			input.inputOptions,
+		);
+	},
+	guessInfo: (input: GuessInfoInput): GuessedInfo | null => {
```

Comment:
@cursoragent No need for guesInfo, as it summons a minion, it doesn't add it to your hand

Reply:
@cursoragent It's likely you're the one with an outdated version. Looking at the "files changed" on the GitHub website, I still see the guessInfo method